canonical-websites/snapcraft.io
# Handle errors public api calls

# Refactor s02e02 - Public protector

## Summary

-  Handle errors for homepage and store page for featured snaps
-  Handle errors for snap store
-  Handle errors for metrics page
- Handle errors for search without illegal characters
- Handle timeouts

## QA

- `./run`
- http://0.0.0.0:8004/ 
- http://0.0.0.0:8004/toto
- http://0.0.0.0:8004/qsdfsqdf
- http://0.0.0.0:8004/store
- http://0.0.0.0:8004/search?q=%3A%3A 
All should work as usual
